packages/vendor-dev/package.json

```diff
@@ -24,6 +24,7 @@
     "eslint.extends.js"
   ],
   "dependencies": {
+    "@testing-library/react": "^8.0.7",
```

Comment:
Sorry about the confusion but the `vendor-dev` package **is not** foreman dev dependencies so `@testing-library/react` should live inside `foreman` dev-dependencies.


While `@theforeman/vendor` gives you an environment to use inside `webpack`, `@theforeman/vendor-dev` should make the same enviorment to work outside of `webpack`. It give you the `vendor` modules to use in `jest`, `eslint`, `storybook`...

Reply:
ok, thanks for having patience with me, I reverted back